### DISCUSSION TITLE
error while loading shared libraries: libdtkwm.so.5

### DISCUSSION THREAD

**Discussion by walrussi** [linuxdeepin/developer-center#5817] [Troubleshooting | 故障排查]
Hi.

deepin-screenshot used to work but suddenly this error appears:

`➜  ~ deepin-screenshot
deepin-screenshot: error while loading shared libraries: libdtkwm.so.5: cannot open shared object file: No such file or directory`

AUR and Pacman cant find libdtkwm and researching couldnt help me either. The AUR [sites](https://archlinux.org/packages/community/x86_64/dtkwm/files/) seem to be offline.

Anything I am missing?

Thanks.

**Comment by zgldh** [comment]
I found the `libdtkwm.so.5` here https://altlinux.pkgs.org/sisyphus/classic-x86_64/libdtkwm-2.0.12-alt1.x86_64.rpm.html

**Comment by BLumia** [comment]
A quick FYI, deepin-screenshot is deprecated and it seems Arch Linux already dropped this package. `deepin-screen-recorder` is the replacement package which provides both screenshot and screen record ability :)

**Reply by zgldh** [reply]
Thank you for your suggestion. But I still love the deepin-screenshot :)
